lib/solidus_paypal_commerce_platform/environment.rb

```diff
@@ -40,8 +40,15 @@ def env_class
     def env_domain
       env.live? ? "www.paypal.com" : "www.sandbox.paypal.com"
     end
+
+    attr_accessor :partner_id, :partner_client_id, :nonce
   end
 
   extend Environment
+
+  # Set some defaults
+  self.nonce = ENV['PAYPAL_NONCE']
+  self.partner_id = ENV['PAYPAL_PARTNER_ID']
```

Comment:
Should these be configurable? Based on Hais comments about the partner information, the impression I got was that this information should not be configurable, because in this instance Nebulab servers as the marketplace provider.

Everything else looks perfect, the extra specs are a nice touch too 👍

Reply:
Also, I'm not sure there's any advantage to making the nonce configurable, AFAIK it doesn't matter what the nonce is, as long as it's a 40 character long string. This is based off of a discussion during my first meeting with PayPal though, so I might be a bit hazy on that.

Reply:
Changed it and added defaults, I left the nonce configurability in place since it's called "Seller-Nonce" and would make sense for it to be different for each seller (is that the same as Partner in this context? 🤔🤷‍♂️).